pkg/storage/storage.go

```diff
@@ -159,24 +167,12 @@ func (s *storageModule) initialize(policy pkg.StoragePolicy) error {
 		return err
 	}
 
-	for _, volume := range existingPools {
-		if _, mounted := volume.Mounted(); mounted {
-			log.Debug().Msgf("Volume %s already mounted", volume.Name())
-			// volume is already mounted, skip mounting it again, make sure it is
-			// in the list of available pools. Since we are in the initialize method,
-			// we can safely assume the pool has not been added before, so no need
-			// to check for duplicate entries.
-			s.volumes = append(s.volumes, volume)
-			continue
-		}
-		_, err = volume.Mount()
+	for _, pool := range existingPools {
+		_, err := pool.Mount()
```

Comment:
Is Mount now idempotent and check if it is already mounted automatically ?

Reply:
I don't understand your question